### ISSUE TITLE
Harvestable observers run once for each quantity instead of once period

### ISSUE THREAD

**Issue by westonleavitt** [westonleavitt/TheOccultBugTracker#238]
![image](https://github.com/westonleavitt/TheOccultBugTracker/assets/99469587/48656f2f-c2c5-45c3-bd01-951ce86af691)

this code causes observer code to be called multiple times during a single harvest interaction. It will call it once for each time you hit a rock. If the tool is a high power it can get called up to 100 times in the same frame

https://streamable.com/g9a0te

**Comment by nbaggett** [comment]
I don't think this is an issue anymore, the current code should only fire off one rpc to sync the inventory at the end of the loop. spawnBaseItem is only called one time.
